Source/WebCore/DerivedSources.make

```diff
@@ -1992,6 +1975,9 @@ MODERN_MEDIA_CONTROLS_SCRIPTS = \
     $(WebCore)/Modules/modern-media-controls/controls/airplay-placard.js \
     $(WebCore)/Modules/modern-media-controls/controls/invalid-placard.js \
     $(WebCore)/Modules/modern-media-controls/controls/pip-placard.js \
+	$(WebCore)/Modules/modern-media-controls/controls/vision-slider.js \
+	$(WebCore)/Modules/modern-media-controls/controls/vision-media-controls.js \
+	$(WebCore)/Modules/modern-media-controls/controls/vision-layout-traits.js \
```

Comment:
same as above, or is GitHub showing me things wrong?

Reply:
ditto